microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -1,235 +1,151 @@
         Works offline and in low-network conditions
     :::column-end:::
     :::column:::
+        :::image type="icon" source="./media/i_progressive-small.png":::
+
+        **[Progressive][MDNPwaAdvantagesProgressive]**  
         Experience scales up (or down) with device capabilities
     :::column-end:::
     :::column:::
+        :::image type="icon" source="./media/i_security-small.png":::
+
+        **[Safe][MDNPwaAdvantagesSafe]**  
         Provides a secure HTTPS endpoint and other user safeguards
     :::column-end:::
 :::row-end:::
 :::row:::
     :::column:::
         :::image type="icon" source="./media/i_responsive-small.png":::
-    :::column-end:::
-    :::column:::
-        :::image type="icon" source="./media/i_link-small.png":::
-    :::column-end:::
-    :::column:::
-        &nbsp;
-    :::column-end:::
-:::row-end:::
-:::row:::
-    :::column:::
-        ### [Responsive][MDNPwaAdvantagesResponsive]
-    :::column-end:::
-    :::column:::
-        ### [Linkable][MDNPwaAdvantagesLinkable]
-    :::column-end:::
-    :::column:::
-        &nbsp;
-    :::column-end:::
-:::row-end:::
-:::row:::
-    :::column:::
+
+        **[Responsive][MDNPwaAdvantagesResponsive]**  
         Adapts to the user's screen size or orientation and input method
     :::column-end:::
     :::column:::
+        :::image type="icon" source="./media/i_link-small.png":::
+
+        **[Linkable][MDNPwaAdvantagesLinkable]**  
         Share and launch from a standard hyperlink
     :::column-end:::
     :::column:::
-        &nbsp;
     :::column-end:::
 :::row-end:::
 
-Build \(or convert\) your existing website to a PWA to enhance engagement with your users. Enhancements include push notifications, app-like integration, and offline support. Continue to build your audience on the open web for users to discover your PWA through search and link-sharing. Best of all, your app is updated using your web server code.
 
-## PWAs on Microsoft Edge (Chromium)
+## PWAs on desktop
 
-When you build a Progressive Web App targeting web standard APIs, your app may be deployed across platforms and devices and take advantage of the device-specific capabilities as available. PWAs in Microsoft Edge \(Chromium\) add the following advantages to your website.
+PWAs aren't only a way for websites to appear on mobile home screens. They're based on standard, cross-browser, web technologies that allow them to be installed and to run everywhere.
 
-*   Your app is built on a standards-based web platform.
-*   Allows your users to install your app directly from the browser.
-*   Allows your users to install your app without a Store-based deployment or registration.
+While most consumers have migrated to mobile-first ways of consuming content, businesses still mostly operate on desktop. Building your desktop app as a PWA allows you to make use of all the benefits that PWAs come with while targeting those users.
```

Comment:
I'm not sure about this framing - I might try to word it slightly differently to avoid getting in a desktop-first vs mobile-first war. Maybe something like:

"In recent years, desktop browsing has grown, and desktop computing continues to be the primary productivity environment for many users. Fortunately, PWAs can provide fully tailored, installable experiences on a desktop operating system that inherit the benefits of that environment, while still working across platform from the same code and servers that power your in-browser and mobile experiences."